src/accounts/pages/Login.tsx

```diff
@@ -204,18 +201,15 @@ export default function Login() {
   }
 
   // creates SSO login request and sends the login request to Tonomy ID, via URL or communication server
-  async function loginToTonomyAndSendRequests(
-    loggedIn = false,
-    user?: ExternalUser
-  ) {
+  async function loginToTonomyAndSendRequests(user?: ExternalUser) {
     try {
       const externalLoginRequest = await UserApps.onRedirectLogin();
 
       getAppDetails(externalLoginRequest);
       const requests = [externalLoginRequest];
       let loginToCommunication: AuthenticationMessage;
 
-      if (loggedIn === false) {
+      if (!user) {
         const { loginRequest, loginToCommunication: loginToCommunicationVal } =
           (await api.ExternalUser.loginWithTonomy({
```

Comment:
before redirect don't we need to logout user?

Reply:
no I don't think so. only if user clicks to logout (`onLogout()`)